test/functorch/test_eager_transforms.py

```diff
@@ -3404,7 +3404,7 @@ def f(x):
     @onlyCPU
     def test_no_warning_on_import_functorch(self, device):
         out = subprocess.check_output(
-            [sys.executable, "-W", "all", "-c", "import functorch"],
+            [sys.executable, "-W", "always", "-c", "import functorch"],
```

Comment:
I guess this is just an unrelated bugfix?

Reply:
`-Wall` is aliased to `-Walways` ([Python docs](https://docs.python.org/3/using/cmdline.html#cmdoption-W)). They are the same.

Reply:
Why this change then?